### PR TITLE
[FIX] pos_{self_order, restaurant}: create missing data for

### PR DESCRIPTION
**Pull request by yoba-odoo**
This commit for fixing the failure of  in No demo build and Multi-build in community. Tour was introduce in https://github.com/odoo/odoo/pull/162119/commits/832fae8dc2c879cfd1dfb57cfdeb5a5c0c3eebef

### CODE REVIEW

**Comment by yoba-odoo** [comment]
@robodoo up to saas-17.2

File: addons/point_of_sale/static/tests/tours/helpers/TicketScreenTourMethods.js [hunk 1]
```diff
@@ -226,7 +226,7 @@ export function receiptTotalIs(amount) {
 export function nthColumnContains(nRow, nCol, string){
     return [
         {
-            trigger: `.ticket-screen .order-row:nth-child(${nRow}) > .col:nth-child(${nCol}):contains("${string}")`,
+            trigger: `.ticket-screen .order-row:nth-last-child(${nRow}) > .col:nth-child(${nCol}):contains("${string}")`,
```
Comment (adgu-odoo) [inline addons/point_of_sale/static/tests/tours/helpers/TicketScreenTourMethods.js:229]: I think you do not have to change this. It does not make much sense to start counting from the end and now that only test orders are shown, you can start counting from the start without problems

File: addons/pos_restaurant/static/tests/tours/TicketScreen.tour.js [hunk 2]
```diff
@@ -76,10 +76,10 @@ registry.category("web_tour.tours").add("OrderNumberConflictTour", {
...
-            TicketScreen.nthColumnContains(6, 2, "Order"),
-            TicketScreen.nthColumnContains(6, 3, "01"),
-            TicketScreen.nthColumnContains(7, 2, "Self-Order"),
-            TicketScreen.nthColumnContains(7, 3, "S"),
-            TicketScreen.nthColumnContains(7, 3, "01")
+            TicketScreen.nthColumnContains(2, 2, "Order"),
+            TicketScreen.nthColumnContains(2, 3, "1"),
+            TicketScreen.nthColumnContains(1, 2, "Self-Order"),
+            TicketScreen.nthColumnContains(1, 3, "S"),
+            TicketScreen.nthColumnContains(1, 3, "1")
```
Comment (adgu-odoo) [inline addons/pos_restaurant/static/tests/tours/TicketScreen.tour.js:83]: you thus will have to change this to something like 
```suggestion
            TicketScreen.nthColumnContains(2, 2, "Order"),
            TicketScreen.nthColumnContains(2, 3, "1"),
            TicketScreen.nthColumnContains(3, 2, "Self-Order"),
            TicketScreen.nthColumnContains(3, 3, "S"),
            TicketScreen.nthColumnContains(3, 3, "1")
```

**Review by adgu-odoo — CHANGES_REQUESTED** [review]

**Comment by Williambraecky** [comment]
Hi 👋 

@yoba-odoo @adgu-odoo can we please move forward with this fix? :)

Thanks

**Comment by yoba-odoo** [comment]
Hey @Williambraecky 

Oups my bad. I am on it.

**Comment by Williambraecky** [comment]
FYI, I started a manual no demo build to make ensure the fix is valid :)

**Comment by Williambraecky** [comment]
Hello 👋 
Where are we on this? It's the last fix to make no demo green in 17.0 (see: https://runbot.odoo.com/runbot/build/70687436)

@yoba-odoo @adgu-odoo

**Comment by yoba-odoo** [comment]
Hello @Williambraecky 

Of course I don't mind you can do this. I am only waiting for @adgu-odoo review if he has any other comments.

**Comment by Williambraecky** [comment]
I was merged today 😇

**Pull request closed without merging**